lib/osf-components/addon/components/node-description/component.ts

```diff
@@ -10,4 +11,22 @@ import template from './template';
 @layout(template, styles)
 export default class NodeDescription extends Component {
     manager!: DescriptionManager;
+    shouldTruncate: boolean = false;
+    truncateDescription: boolean = true;
+
+    setTruncate(element: Element, args: any[]) { // args: [description, this]
```

Comment:
Ideally, we never ever use `any`. You can explicitly type the elements of an array (and use destructuring to avoid the awkward `args[1]`) like this:
```ts
setTruncate(element: Element, [description, node]: [string, NodeDescription]) {
```
But, as mentioned below (or above?), we should bind the function instead of passing `this`.